eng/test-job.yml

```diff
@@ -203,6 +203,31 @@ jobs:
           - jitstress2_tiered
           - zapdisable
           - tailcallstress
+        ${{ if eq(parameters.testGroup, 'outerloop-jitstressisas') }}:
+          scenarios:
+          - jitstressisas_incompletehwintrinsic
+          - jitstressisas_nohwintrinsic
+          - jitstressisas_nohwintrinsic_nosimd
+          - jitstressisas_nosimd
+        ${{ if and(eq(parameters.testGroup, 'outerloop-jitstressisas'), or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86'))) }}:
```

Comment:
```suggestion
          ${{ if or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86')) }}:
```

Reply:
And indent the rest, might be worth a try, although not sure if it will work.

Reply:
That's what I was going to attempt first. If that doesn't work, I can just duplicate the four "shared" jobs in an ARM/ARM64 specific grouping